float8_experimental/float8_ops.py

```diff
@@ -50,7 +50,38 @@ def float8_desugar_op(aten_op, args, kwargs=None):
     )
 
 
-@implements([aten.sum.dim_IntList])
+@implements([aten.split.Tensor])
+def float8_split(aten_op, args, kwargs=None):
+    new_data_tensors = aten_op(args[0]._data, *args[1:], **kwargs)
+
+    def make_float8(data):
+        return Float8Tensor(
+            data, args[0]._scale, args[0]._orig_dtype, args[0]._mm_config
+        )
+
+    out = map(make_float8, new_data_tensors)
+    return list(out)
+
+
+# Errors cant `cat_cuda float8 e4m3fn`
```

Comment:
oh so this means that the `torch.cat` can't apply to dtype e4m3fn?

Normally I feel this is something that we can just make our cuda kernel to support concatting tensors with the same dtype, but not sure if there're further complications there for fp8 dtype.

But if the job we want to do is to simply concatting the fp8 tensors together, one simpler way to can do:

we can just try to do `fp8_inner_tensor.view(torch.uint8)`, perform `torch.cat`, then after the cat operation, we do `fp8_catted_tensor.view(torch.float8_e4m3fn`, I wonder if this would unblock?